issuance/issuance.go

```diff
@@ -76,13 +76,13 @@ type IssuerConfig struct {
 // Only one of File, ConfigFile, or PKCS11 should be set.
 type IssuerLoc struct {
 	// A file from which a private key will be read and parsed.
-	File string
+	File string `validator:"required"`
```

Comment:
`File` is only required if `ConfigFile` is not present (in fact, it's basically deprecated though not marked as so).

I'm surprised this passed tests, since our actual CA configs don't set `File`. Maybe the `Location` field of `IssuerConfig` needs `dive`? Or the layer above that?

Reply:
You found two different issues here:

1. I wrote `validator:` instead of `validate:` 🤦🏻‍♀️
1. I wasn't `dive`ing each of the slices and I should have been.

These are now fixed here and elsewhere.